src/main/java/org/springframework/integration/aws/inbound/SqsMessageDrivenChannelAdapter.java

```diff
@@ -88,6 +89,10 @@ public void setResourceIdResolver(ResourceIdResolver resourceIdResolver) {
 		this.simpleMessageListenerContainerFactory.setResourceIdResolver(resourceIdResolver);
 	}
 
+	public void setAutoStartUp(boolean autoStartUp) {
+		super.setAutoStartup(autoStartUp);
```

Comment:
Apologies, I realise I was trying to `@Override`, `setAutoStartUp` and not `setAutoStartup`.